lib/validators/cms_schematron_validator.rb

```diff
@@ -7,7 +7,7 @@ class CMSSchematronValidator < QrdaFileValidator
     @bundle_version = APP_CONFIG.default_bundle
 
     def initialize(schematron_file, name, bundle_version = APP_CONFIG.default_bundle)
-      @validator = Schematron::Validator.new(name, schematron_file) if File.exists?(schematron_file)
+      @validator = Schematron::Validator.new(name, schematron_file) if File.exist?(schematron_file)
```

Comment:
Note this change is unrelated, but was made because rubocop said `File.exists?` was deprecated